modules/management-console-agent/src/test/java/org/apache/ignite/agent/action/controller/QueryActionsControllerTest.java

```diff
@@ -365,7 +453,7 @@ public void shouldReturnRunningQueriesFromAllNodes() {
                         .setQueryId(UUID.randomUUID().toString())
                         .setQueryText("SELECT count(*), sleep() AS \"" + nid + "\" FROM \"TestCache\".STRING")
                         .setPageSize(1)
-                        .setCacheName("TestCache")
+                        .setDefaultSchema("TestCache")
```

Comment:
Let's add test where schema is removed from query and set we set default schema

Reply:
This variant is already implemented in my test:
getSelectQuery(fullTestTable) + getSelectQuery(testTable)